src/test/ui/while-let.stderr

```diff
@@ -13,10 +13,10 @@ LL | |     });
    = note: this warning originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
 
 warning: irrefutable while-let pattern
-  --> $DIR/while-let.rs:7:13
+  --> $DIR/while-let.rs:7:19
    |
 LL |               while let $p = $e $b
-   |               ^^^^^^^^^^^^^^^^^^^^
+   |                     ^^^
```

Comment:
This doesn't look right.

Reply:
Spans involving macros are sometimes less than ideal :-/